.ebextensions/01_create_apache_conf.config

```diff
@@ -126,7 +126,6 @@ files:
         SSLSessionTickets     Off
 
         Header always set Strict-Transport-Security "max-age=63072000; includeSubdomains; preload"
-        Header always set X-Frame-Options DENY
```

Comment:
@paulkerschen : Not having the X-Frame_Options resulted in few errors on the LRS side of things. I used this as a way to get around the config.

"Header always unset X-Frame-Options"